### PR TITLE
wv-1983 Night Lights Tour Story

### PR DESCRIPTION
**Pull request by minniewong**
## Description

Fixes WV-1983

Night Lights Tour Story using Black Marble At-Sensor Radiance and Blue/Yellow Composite.

@nasa-gibs/worldview

### CODE REVIEW

**Review by jwhurley1 — APPROVED** [review]
Looks great!  Very cool tour story!

**Comment by minniewong** [comment]
> Step 4 says, "(A different color palette was applied to this comparison to highlight the differences between the two images.)" but when I switch between A and B I see the same palette selected 🤔

I guess I should word that better? the layer on both sides do have the same color palette, but it's different from the "normal" greyscale colours

**Comment by jasontk19** [comment]
> > Step 4 says, "(A different color palette was applied to this comparison to highlight the differences between the two images.)" but when I switch between A and B I see the same palette selected 🤔
> 
> I guess I should word that better? the layer on both sides do have the same color palette, but it's different from the "normal" greyscale colours

I think it's OK, probably only going to confuse people like me :)

**Review by jasontk19 — APPROVED** [review]
Looks good to me!

**Comment by rboller** [comment]
> > > Step 4 says, "(A different color palette was applied to this comparison to highlight the differences between the two images.)" but when I switch between A and B I see the same palette selected 🤔
> > 
> > 
> > I guess I should word that better? the layer on both sides do have the same color palette, but it's different from the "normal" greyscale colours
> 
> I think it's OK, probably only going to confuse people like me :)

Actually that's also confusing to me so would suggest rewording it.

File: config/default/common/config/metadata/stories/black_marble_night_lights/step001.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The Day/Night Band allows scientists to monitor daily various nighttime phenomena on the Earth's surface and atmosphere in low light conditions. This Black Marble Nighttime At Sensor Radiance layer is created from the Day/Night Band-based at-sensor top-of-atmosphere night time radiance product (VNP46A1).
```
Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step001.md:1]: --> "The Day/Night Band allows scientists to monitor various nighttime phenomena on the Earth's surface and atmosphere in low light conditions on a daily basis."

Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step001.md:1]: I'm struggling to parse "... is created from Day/Night Band-based at-sensor top-of-atmosphere night time radiance product (VNP46A1)."

File: config/default/common/config/metadata/stories/black_marble_night_lights/step004.md [hunk 2]
```diff
@@ -0,0 +1 @@
+A major power outage occurred in Houston, Texas on 16 February 2021 due to severe winter weather. On the left is an image from before the storm on 7 February 2021, the city lights fill up most of the central area and emanates outwards along major roads. Move the center swipe line to the left and dark patches appear in the western, eastern, and southern portions of the city on 16 February 2021 where power outages affected the city for several days. In this comparison, we've applied a different color palette to highlight the differences between the two images. You can change the color palette by clicking on the "View Options" icon on the layer and select from a range of different color palettes.
```
Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step004.md:1]: "emanates" --> "emanate"

File: config/default/common/config/metadata/stories/black_marble_night_lights/step005.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The Black Marble Blue/Yellow Composite (Day/Night Band) layer is a false color composite using the VIIRS at-sensor radiance and the brightness temperatures from the M15 band and complements the At Sensor Radiance layer. This false color scheme produces nighttime city lights in shades of yellow with infrared, nighttime cloud presence in shades of blue. During bright moonlight conditions, moonlight reflected from cloud tops and the land surface may also provide a yellow hue to those features. This image shows Tropical Storm Mindulle on 30 September 2021 nearing Japan. [Learn more about the Blue/Yellow composite](https://earthdata.nasa.gov/learn/articles/black-marble-blue-yellow-composite)
```
Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step005.md:1]: --> "The Black Marble Blue/Yellow Composite (Day/Night Band) layer is a false color composite using VIIRS at-sensor radiance combined with brightness temperature from the M15 band which complements the At Sensor Radiance layer."

File: config/default/common/config/metadata/stories/black_marble_night_lights/step006.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Besides being able to see anthropogenic light sources from space, this swirling streak of light over the Hudson Bay is actually an aurora borealis/northern lights event on 4 February 2022.
```
Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step006.md:1]: We can link to can link to the related Earth Observatory story for more info: https://earthobservatory.nasa.gov/images/149435/brilliant-borealis

**Review by rboller — CHANGES_REQUESTED** [review]
Nice examples of a really neat layer.  I've requested a few updates for clarity but otherwise think it looks great.

File: config/default/common/config/metadata/stories/black_marble_night_lights/step001.md (hunk 1 above)
Comment (minniewong) [reply config/default/common/config/metadata/stories/black_marble_night_lights/step001.md:1]: i was struggling with that one for some reason! thanks for making it better :D

File: config/default/common/config/metadata/stories/black_marble_night_lights/step006.md (hunk 4 above)
Comment (minniewong) [reply config/default/common/config/metadata/stories/black_marble_night_lights/step006.md:1]: I had refrained from adding it because it looks so much better in the article, but will add it in.

File: config/default/common/config/metadata/stories/black_marble_night_lights/step004.md [hunk 5]
```diff
@@ -0,0 +1 @@
+A major power outage occurred in Houston, Texas on 16 February 2021 due to severe winter weather. On the left is an image from before the storm on 7 February 2021, the city lights fill up most of the central area and emanate outwards along major roads. Move the center swipe line to the left and dark patches appear in the western, eastern, and southern portions of the city on 16 February 2021 where power outages affected the city for several days. In this comparison, we've applied a different color palette to highlight the differences between the two images. You can change the color palette by clicking on the "View Options" icon on the layer and select from a range of different color palettes.
```
Comment (rboller) [inline config/default/common/config/metadata/stories/black_marble_night_lights/step004.md:1]: --> "In this comparison, we've applied a different color palette to highlight the differences between the brighter and darker areas."

**Review by rboller — APPROVED** [review]
The updates look great to me!
